modules/handle_state.js

```diff
@@ -0,0 +1,19 @@
+const { Request } = require('./api');
+
+module.exports.CardOn = async (win, uuid) => {
+  await Request('GET', `api/checkout/card?uuid=${uuid}`, null, (err, data, statusCode) => {
```

Comment:
Should be ok statuscode first and any error codes afterwards. If you get a statusCode such as 500 / internal server error you still progress to cart which shouldn't happen